### PR TITLE
docs: loading tips update

### PR DESCRIPTION
**Pull request by kpotschi**
## Summary of Changes

Updated the number of produced A320 to the latest available number, based on Airbus website.
Tiny grammar syntax correction of German Loading Tips

## References

https://en.wikipedia.org/wiki/Airbus_A320_family#cite_note-Airbus_Orders-5
https://www.airbus.com/aircraft/market/orders-deliveries.html

## Additional context

Discord username (if different from GitHub):
MelodyConnor#2805

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by kpotschi** [comment]
First try in using Github to participate. Feedback welcome!

**Comment by Phil7789** [comment]
88d786d
Correction of the german loading tips is unnecessary as it was correct before.

**Comment by kpotschi** [comment]
It didn't necessarily strike me as "wrong" but a little unclear about whether the fbw came first or the a320 came first.

File: flybywire-aircraft-a320-neo/de-DE.locPak [hunk 1]
```diff
@@ -9,10 +9,10 @@
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Ausgewählten Steuerkurs nach rechts anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
       "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU verfügbar",
-      "A32NX.LOADING.TIPS.A320NEO_001": "Der FlyByWire A32NX ist das Ergebnis der Zusammenarbeit von Hunderten von individuellen Mitarbeitern.",
+      "A32NX.LOADING.TIPS.A320NEO_001": "Der FlyByWire A32NX ist das Ergebnis der Zusammenarbeit von Hunderten von individuellen Mitwirkenden.",
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/de-DE.locPak:12]: Wenn wir schon einmal dabei sind hier was zu ändern dann bitte nur einmal den genetiv benutzen.

Außerdem finde ich wahl `Mitwirkende` komisch, aber es wäre die korrekte übersetzung.

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_001": "Der FlyByWire A32NX ist das Ergebnis der Zusammenarbeit von Hunderten individuellen Mitwirkenden.",
```

File: flybywire-aircraft-a320-neo/de-DE.locPak [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
       "A32NX.LOADING.TIPS.A320NEO_002": "Wussten Sie schon? Der A320 kann bei Seitenwinden bis zu 38 Knoten und Rückenwinden bis zu 15 Knoten landen!",
-      "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire ist nach der Fly-by-Wire-Steuerung benannt, die erstmals im Airbus A320 eingeführt wurde.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "Es sind 9693 Flugzeuge der A320-Familie gebaut worden! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
+      "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire ist nach der Fly-by-Wire-Steuerung benannt, die im Airbus A320 erstmals eingeführt wurde.",
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/de-DE.locPak:14]: Weder müssen wir den satz hier teilen noch einen artikel zweckentfremden, ein pronomen passt hier sogar besser.

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire ist nach der Fly-by-Wire-Steuerung benannt welche erstmals im Airbus A320 verwendet wurde.",
```

File: flybywire-aircraft-a320-neo/de-DE.locPak [hunk 3]
```diff
@@ -9,10 +9,10 @@
...
+      "A32NX.LOADING.TIPS.A320NEO_004": "Es sind 9873 Flugzeuge der A320-Familie gebaut worden! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/de-DE.locPak:15]: Verstehe diese vermischung von perfekt und plusquamperfekt nicht.

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "Es wurden bislang 9873 Flugzeuge der A320-Familie gebaut! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
```

Macht es nicht mehr sinn, eine sich stätig ändernde zahl anders zu präsentieren?

Und dann später beinahme mit über tauschen sobald es mehr sind?

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "Es wurden schon beinahe 10000 Flugzeuge der A320-Familie gebaut! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
```

File: flybywire-aircraft-a320-neo/en-US.locPak [hunk 4]
```diff
@@ -16,7 +16,7 @@
       "A32NX.LOADING.TIPS.A320NEO_001": "The FlyByWire A32NX is a collaboration between hundreds of individual contributors.",
       "A32NX.LOADING.TIPS.A320NEO_002": "Did you know? The A320 can land at crosswinds up to 38 knots and tailwinds of up to 15 knots!",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire is named after the fly-by-wire controls first introduced in the Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "There are 9693 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
+      "A32NX.LOADING.TIPS.A320NEO_004": "There are 9873 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/en-US.locPak:19]: One could do the same thing as proposed in german above.

switch the logic to not use an single number for a constantly changing value, use a range instead.

and once passed 10k we change it to be more than 10K. This way the value is never really outdated and we don't need to constantly update production numbers.

```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "There are almost 9873 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
```

File: flybywire-aircraft-a320-neo/es-ES.locPak [hunk 5]
```diff
@@ -12,7 +12,7 @@
       "A32NX.LOADING.TIPS.A320NEO_001": "El FlyByWire A32NX es una colaboración entre cientos de colaboradores individuales.",
       "A32NX.LOADING.TIPS.A320NEO_002": "¿Sabías que? El A320 puede aterrizar con vientos cruzados de hasta 38 nudos y con vientos de cola de hasta 15 nudos.",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire recibe su nombre de los controles fly-by-wire introducidos por primera vez en el Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "Hay 9693 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
+      "A32NX.LOADING.TIPS.A320NEO_004": "Hay 9873 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
```
Comment (MisterChocker) [inline flybywire-aircraft-a320-neo/es-ES.locPak:15]: same can be done here as well. and for all the other languages.

**Review by MisterChocker — CHANGES_REQUESTED** [review]
So I've made some suggestions to the german proposal and one general proposal to how to better handle production numbers.

The german part is in german. Sorry for all not germans out there!

**Comment by Phil7789** [comment]
> Wenn wir schon einmal dabei sind hier was zu ändern dann bitte nur einmal den genetiv benutzen.
> 
> Außerdem finde ich wahl `Mitwirkende` komisch, aber es wäre die korrekte übersetzung.

As "Mitarbeiter" is indeed a valid translation for contributor there is no need for change imo as it was correct in the first place and I think it reflects the labor done during the building process a bit better (while not forgetting the engineers inventing the plane).
The other changes are good improvements.

To not change the code back and forth again: How about using "more than" as the number of aircrafts build? We could reference a number from the past while keeping validity for the future.

**Comment by MisterChocker** [comment]
As "Mitarbeiter" is indeed a valid translation for contributor there is no need for change imo as it was correct in the first place

No it's not `Mitarbeiter` means employee and not contributor.

**Comment by kpotschi** [comment]
> As "Mitarbeiter" is indeed a valid translation for contributor there is no need for change imo as it was correct in the first place and I think it reflects the labor done during the building process a bit better (while not forgetting the engineers inventing the plane).

I think what made me challenge it in the first place was the "von individuellen Mitarbeitern" as in - when is a person not individual?

> The other changes are good improvements.

agreed.

> To not change the code back and forth again: How about using "more than" as the number of aircrafts build? We could reference a number from the past while keeping validity for the future.

I think that's a good idea. We just gotta keep Tuvalu in mind in the future :)

**Comment by Phil7789** [comment]
> I think what made me challenge it in the first place was the "von individuellen Mitarbeitern" as in - when is a person not individual?

Yeah I see that. It could also be translated to "von Hunderten einzelner Mitarbeiter"

**Comment by MisterChocker** [comment]
Why not just use `Der FlyByWire A32NX ist das Ergebnis der Zusammenarbeit von Hunderten luftfahrtbegeisterten`

File: flybywire-aircraft-a320-neo/en-US.locPak (hunk 4 above)
Comment (ErickSharp) [reply flybywire-aircraft-a320-neo/en-US.locPak:19]: I have to disagree; `about` would probably be a better word as `almost` signifies that the real number is approaching that value, but has not approached it.

File: flybywire-aircraft-a320-neo/es-ES.locPak (hunk 5 above)
Comment (ErickSharp) [inline flybywire-aircraft-a320-neo/es-ES.locPak:15]:
```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "Hay cerca de 9873 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
```

**Review by ErickSharp — CHANGES_REQUESTED** [review]

**Comment by kpotschi** [comment]
Commit 0bbc132 now has the updated logic to it. "More than 9900...".

I had all changes in the different languages reviewed by native speakers, that's why it took a moment.

**Comment by Rapx3** [comment]
In French, you have to had a space between a word and an exclamation mark.
So 
> "Plus de 9900 avions de la famille A320 ont été construits!

should become

> "Plus de 9900 avions de la famille A320 ont été construits !

**Comment by kpotschi** [comment]
> In French, you have to had a space between a word and an exclamation mark.
> So
> 
> > "Plus de 9900 avions de la famille A320 ont été construits!
> 
> should become
> 
> > "Plus de 9900 avions de la famille A320 ont été construits !

interesting, will correct that! thanks

**Review by MisterChocker — APPROVED** [review]
German LGTM!

File: flybywire-aircraft-a320-neo/es-ES.locPak (hunk 5 above)
Comment (kpotschi) [reply flybywire-aircraft-a320-neo/es-ES.locPak]: "¡Se han construido más de 9900 aviones de la familia A320!" is what i became

**Comment by MisterChocker** [comment]
> > In French, you have to had a space between a word and an exclamation mark.
> > So
> > > "Plus de 9900 avions de la famille A320 ont été construits!
> > 
> > 
> > should become
> > > "Plus de 9900 avions de la famille A320 ont été construits !
> 
> interesting, will correct that! thanks

@kpotschi can you do that?

**Comment by kpotschi** [comment]
> > > In French, you have to had a space between a word and an exclamation mark.
> > > So
> > > > "Plus de 9900 avions de la famille A320 ont été construits!
> > > 
> > > 
> > > should become
> > > > "Plus de 9900 avions de la famille A320 ont été construits !
> > 
> > 
> > interesting, will correct that! thanks
> 
> @kpotschi can you do that?

done

**Pull request merged**